src/Faithlife.Tracing/AspNet/AspNetTracing.cs

```diff
@@ -0,0 +1,91 @@
+#if NET461
+using System;
+using System.Threading;
+using System.Web;
+
+namespace Faithlife.Tracing.AspNet
+{
+	public static class AspNetTracing
+	{
+		public static void BeginTracing(string serviceName)
+		{
+			var httpContext = HttpContext.Current;
+			if (httpContext == null)
+				return;
+
+			var headers = httpContext.Request.Headers;
+			var b3Sampled = headers[TracingUtility.B3SampledHeaderName];
+			if (string.Equals(b3Sampled, "0"))
```

Comment:
Why `string.Equals`?

Reply:
Because I forgot the equality operator existed.